hypervisor/src/hypervisor/partition.rs

```diff
@@ -275,6 +277,24 @@ impl Run {
                 //umount("old").unwrap();
                 chdir("/").unwrap();
 
+                // After we've performed the pseudo chroot, we can create the
+                // Unix domain socket
+                let syscall_socket = socket::socket(
+                    AddressFamily::Unix,
+                    SockType::Datagram,
+                    SockFlag::SOCK_CLOEXEC,
+                    None,
+                )
+                .unwrap();
+
+                bind(syscall_socket, &UnixAddr::new("/syscall").unwrap()).unwrap();
```

Comment:
I would favour a more specific domain socket path. Maybe `/syscall-a653` ? Just to avoid confusion, because we are re-using a term `syscall` which is already established with different semantics than what we use it for.

Reply:
Sounds like a good idea to me! At that point, I would also consider making this a constant then :^)